sdk/go/common/workspace/plugins.go

```diff
@@ -190,10 +191,11 @@ func (info PluginInfo) Download() (io.ReadCloser, int64, error) {
 	// is hosted by Pulumi.
 	serverURL := info.ServerURL
 	if serverURL == "" {
-		serverURL = "https://api.pulumi.com/releases/plugins"
+		serverURL = "https://get.pulumi.com/releases/plugins"
 	}
 
 	endpoint := fmt.Sprintf("%s/pulumi-%s-%s-v%s-%s-%s.tar.gz", serverURL, info.Kind, info.Name, info.Version, os, arch)
+	endpoint = strings.ReplaceAll(endpoint, "//", "/")
```

Comment:
Why is this needed? In case someone specifies a server URL that ends with `/`? If so, why not just trim the end, e.g. `strings.TrimSuffix(serverURL, "/")`?

Reply:
Yes, that's exactly it. If you enter `--server https://ending-with-a-slash/` you get an error. `strings.TrimSuffix` is cleaner and clearer, so I'll update the code to do that instead. Thanks.